04/app.js

```diff
@@ -1,9 +1,31 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: 'Adam',
+	lastName: 'Nowak',
+	born: {
+		day: '2',
+		month: '10',
+		year: '2023',
+		// fullDate: () => {
+		// 	return Object.values(user.born).join('-')
+		// },
+	},
+}
+const isUserBirthday = () => {
+	// get user birthday
+	const userBirthDay = Object.values(user.born).join('-')
```

Comment:
Zdecydowanie lepiej przekazać `user` przez parametr - wtedy funkcja jest bardziej elastyczna.
Obecnie ta funkcja może działać z konkretnym obiektem (`user`), a to w większej aplikacji niezbyt wygodne rozwiązanie ;)